libkpod/remove.go

```diff
@@ -0,0 +1,62 @@
+package libkpod
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/sirupsen/logrus"
+)
+
+// RemoveContainers removes the given containers. If the container is running, the container
+// should be force removed.
+func (c *ContainerServer) RemoveContainers(containers []string, force bool) string {
+	var ret string
+	for _, container := range containers {
+		id, err := c.remove(container, force)
+		if err != nil {
+			ret = append(ret, err.Error(), "\n")
+		} else {
+			ret = append(ret, id, "\n")
+		}
+	}
+	return ret
+}
+
+func (c *ContainerServer) remove(container string, force bool) (string, error) {
+	ctr, err := c.LookupContainer(container)
+	if err != nil {
+		return "", err
+	}
+	ctrID := ctr.ID()
+
+	if err := c.runtime.UpdateStatus(ctr); err != nil {
+		return ctrID, fmt.Errorf("failed to update container state: %v", err)
+	}
+
+	cState := c.runtime.ContainerStatus(ctr)
+	if cState.Status == oci.ContainerStateCreated || cState.Status == oci.ContainerStateRunning {
+		if err := c.runtime.StopContainer(ctr, -1); err != nil {
```

Comment:
I've been trying to put all of the logic into libkpod and basically leave kpod as an I/O layer.  By checking if a container is stopped, we have to look up the container and get the state within the cli, all of which seems like logic that should be in libkpod for others to use if the want to.  I did change the logic to check if the container was stopped though